Codeinwp/neve
# Feat add bf notice

### Summary
Added the bf notice in Neve's dashboard

### Will affect the visual aspect of the product
NO


### Test instructions


- Download this version of [neve.zip](https://github.com/Codeinwp/neve/files/9911113/neve.zip)
- Check the banner in the dashboard



Closes Codeinwp/themeisle#1397.
